### ISSUE TITLE
Composer doesn't work properly

### ISSUE THREAD

**Issue by topwebs** [magento/devdocs#464]
tshoot_composer-fail.md says issues with composer in certain date range. Latest composer of December 22 gives same error, so troubleshooter is wrong. Make install work with latest composer. Your basic magento install instructions for composer don't work.

**Comment by xcomSteveJohnson** [comment]
Please clarify what the problem is. Are you seeing the same error?

**Comment by topwebs** [comment]
Yes.

**Comment by xcomSteveJohnson** [comment]
Please provide the output of `composer -v`

**Comment by topwebs** [comment]
Composer version 1.0-dev (d6ae9a0529e1f39c4c7f9b2f29fff019d79cd1fb) 2015-12-22 20:44:41

**Comment by xcomSteveJohnson** [comment]
We're sorry for the issue you're having. Thanks for your patience. This is my composer version:

`Composer version 1.0-dev (72cd6afdfce16f36a9fd786bc1b2f32b851e764f) 2015-12-28 17:35:19`

Using that, I switched to the Magento file system owner and used the following command to successfully download CE (I did not install it).

`composer create-project --repository-url=https://repo.magento.com/ magento/project-community-edition magento2`

Seems like it should work for you. What are you doing differently?
